services/backend/src/routes/protected_areas.js

```diff
@@ -705,35 +705,5 @@ module.exports = (errorHandler, paService) => {
       }),
     ),
   );
-
-  /**
```

Comment:
route `/pa/:category/coverage` should also be deleted

Reply:
Done!